### PR TITLE
Fix more incorrect entries

### PR DESCRIPTION
**Pull request by luzpaz**

### CODE REVIEW

File: dictionary_all.txt [hunk 1]
```diff
@@ -9838,13 +9841,12 @@ atchach->attach
 atcohot->alcohol
 atcohotic->alcoholic
 atcually->actually
-atcualy->actualy
-ate->eight
```
Comment (luzpaz) [inline dictionary_all.txt:9842]: removed because it's a valid word

File: dictionary_all.txt [hunk 2]
```diff
@@ -9997,7 +9999,6 @@ atlainder->attainder
 atlained->attained
 atlantc->atlantic
 atlantia->atlanta
-atlantis->atlantic
```
Comment (luzpaz) [inline dictionary_all.txt:10000]: removed due to valid word

File: dictionary_all.txt [hunk 3]
```diff
@@ -31323,9 +31324,8 @@ developpers->developers
 developping->developing
 developpment->development
 developps->develops
-developres->develops
-developrs->develops
-develops->developers
```
Comment (luzpaz) [inline dictionary_all.txt:31328]: valid word

**Comment by luzpaz** [comment]
Ready for merge

**Comment by orbitcowboy** [comment]
Thanks!

**Pull request merged**